Multisig/Assets.xcassets/ico-ether.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
Just use single scale instead of individual

Reply:
ether icon images were provided as 3 images